app/api/controllers/validators/validations/Validation.scala

```diff
@@ -14,6 +14,6 @@
  * limitations under the License.
  */
 
-package api.controllers.requestParsers.validators.validations
+package api.controllers.validators.validations
 
 trait Validation
```

Comment:
What is the purpose of this Validation trait? I can see all the validations extend it, but it doesn't seem to be used anywhere else.